docs/how-to-guides/new-user-guides/helm-charts-in-rancher/helm-charts-in-rancher.md

```diff
@@ -66,82 +65,92 @@ Rancher v2.5 replaced the former catalog system with a new **Apps & Marketplace*
 
 Since Rancher v2.6.5, the **Apps & Marketplace** feature is named **Apps** in the UI.
 
-### Charts
+## Charts
 
-From the top-left menu select **Apps** and you will be taken to the **Charts** page.
+The **Charts** page contains all Rancher, Partner, and Custom charts. You can filter charts by selecting the left-most dropdown menu:
 
-The **Charts** page contains all Rancher, Partner, and Custom charts.
+* Rancher tools such as Logging or Monitoring are listed under the **Rancher** label.
+* Partner charts are under the **Partners** label.
+* Custom charts are listed under the name of their respective repository.
 
-* Rancher tools such as Logging or Monitoring are included under the Rancher label.
-* Partner charts reside under the Partners label.
-* Custom charts will show up under the name of the repository.
-
-All three types are deployed and managed in the same way.
+All three types of charts are deployed and managed in the same way.
 
 :::note
 
-Apps managed by the Cluster Manager (the global view in the legacy Rancher UI) should continue to be managed only by the Cluster Manager, and apps managed with **Apps** in the new UI must be managed only by **Apps**.
+Apps managed by the Cluster Manager (the global view in the legacy Rancher UI) continue to be managed only by the Cluster Manager, and apps managed with **Apps** in the new UI must be managed only by **Apps**.
 
 :::
 
-### Repositories
+To access the **Charts** page:
+
+1. Click **☰**. Under **Explore Cluster** in the left navigation menu, select a cluster.
```

Comment:
I think cluster access should go through the **Cluster Management** -> **Clusters** route since that page will have the full list of clusters. I don't think that's the case for the left nav cluster list.

Also, `Explore Cluster` is not a label in the left nav of 2.8.